engine/src/test/java/org/camunda/bpm/engine/test/history/HistoricIncidentQueryTest.java

```diff
@@ -500,6 +499,46 @@ public void testQueryByInvalidConfigurationId() {
     } catch (ProcessEngineException e) { }
   }
 
+  @Test
+  @Deployment(resources={"org/camunda/bpm/engine/test/api/runtime/oneFailingServiceProcess.bpmn20.xml"})
+  public void testQueryByHistoryConfiguration() {
+    startProcessInstance(PROCESS_DEFINITION_KEY);
+
+    // get the latest historic job log id
+    Integer latestJobLogId = historyService.createHistoricJobLogQuery().list().stream().map(hjl -> Integer.parseInt(hjl.getId())).max(Integer::compare).get();
```

Comment:
🔧 I suggest adjusting the formatting here, split the row, that can help for better readability.